pandas/__init__.py

```diff
@@ -36,6 +36,9 @@
 
 from datetime import datetime
 
+from pandas._config.config import (get_option, set_option, reset_option,
```

Comment:
what I mean was to have all of these listed in the 'all' in pandas._config, then this can just be a * import and then you can *always* just ``from pandas._config import .....`` rather than reaching into ``pandas._config.config`` (I know there are some cases of that but can fix those in a follow). this would make clear then this modules public api.

Reply:
OK.  Moved those into `_config.__init__`.  Will update all the other imports on the next pass.  Will ignore the "*" import idea.

Reply:
> Will ignore the "*" import idea.

we do this for all api imports in pandas/__init__.py

Reply:
... for now.